scripts/live_test/CLITest.yml

```diff
@@ -0,0 +1,259 @@
+      maps:
+        Target: maps
+      monitor:
+        Target: monitor
+      natgateway:
+        Target: natgateway
+      netappfiles:
+        Target: netappfiles
+      network:
+        Target: network
+      policyinsights:
+        Target: policyinsights
+      privatedns:
+        Target: privatedns
+      profile:
+        Target: profile
+      rdbms:
+        Target: rdbms
+      redis:
+        Target: redis
+      relay:
+        Target: relay
+      reservations:
+        Target: reservations
+      resource:
+        Target: resource
+      role:
+        Target: role
+      search:
+        Target: search
+      security:
+        Target: security
+      servicebus:
+        Target: servicebus
+      servicefabric:
+        Target: servicefabric
+      signalr:
+        Target: signalr
+      sql:
+        Target: sql
+      sqlvm:
+        Target: sqlvm
+      storage:
+        Target: storage
+      util:
+        Target: util
+      vm:
+        Target: vm
+  pool:
+    vmImage: 'ubuntu-18.04'
+  steps:
+    - bash: |
+        if [[ "$(USER_TARGET)" =~ "$(Target)" || "$(USER_TARGET)" == "" ]]; then
+          echo "Match!"
+          echo "##vso[task.setvariable variable=Match]1"
+        else
+          echo "Not match!"
+          echo "##vso[task.setvariable variable=Match]0"
+        fi
+    - task: UsePythonVersion@0
+      condition: eq(variables.Match, '1')
+      inputs:
+        versionSpec: '3.8'
+        addToPath: true
+        architecture: 'x64'
+    - bash: |
+        cd ..
+        pwd
+        ls
+        # ssh-keygen -f ~/.ssh/id_rsa -t rsa -N ''
+        git clone -b $(USER_BRANCH) $(USER_REPO)
+        #git clone https://github.com/Azure/azure-cli-extensions.git
+        python -m venv env
+        source env/bin/activate
+        # pip install azdev
+        git clone https://github.com/Azure/azure-cli-dev-tools.git
+        pip install -e azure-cli-dev-tools
+        pip install pytest-json-report
+        pip install pytest-html
+        pip install pytest-rerunfailures
+        azdev setup -c azure-cli
+        # az login --service-principal --username $(APP_ID) --password $(PASSWORD) --tenant $(TENANT_ID)
+        az login -u azureclitest@azuresdkteam.onmicrosoft.com -p $(AZURECLITESTPASSWORD)
+        az account set -s 0b1f6471-1bf0-4dda-aec3-cb9272f09590
+        # Sequential
+        azdev test $(Target) $(USER_LIVE) --mark serial --xml-path test_results.sequential.xml --no-exitfirst -a "--json-report --json-report-summary --json-report-file=$(Target).report.sequential.json --html=$(Target).report.sequential.html --self-contained-html --reruns 3 -s"
+        # Parallel
+        azdev test $(Target) $(USER_LIVE) --mark "not serial" --xml-path test_results.parallel.xml --no-exitfirst -a "-n $(USER_PARALLELISM) --json-report --json-report-summary --json-report-file=$(Target).report.parallel.json --html=$(Target).report.parallel.html --self-contained-html --reruns 3 -s"
```

Comment:
why called "not serial" instead of "parallel"?

Reply:
"not serial" is not a plain string. "not" is an operator. If I use "parallel", I need to force users to add marks for all tests. Only a small portion of tests need to be marked as serial.